libdnf/sack/advisory.cpp

```diff
@@ -151,9 +151,66 @@ std::vector<AdvisoryModule> Advisory::getModules() const
         moduleList.emplace_back(sack, advisory, name, stream, version, context, arch);
     }
     dataiterator_free(&di);
+
     return moduleList;
 }
 
+std::vector<AdvisoryPkg>
+Advisory::getApplicablePackages() const
```

Comment:
Please could you consider to add optional argument `bool withFilemanes`. It is a performance optimization. See getPackages(std::vector<AdvisoryPkg> & pkglist, bool withFilemanes) above.

Reply:
Added, thanks. It is used from `filterAdvisory` just like with `getPackages`.